common.yaml

```diff
@@ -21,10 +21,33 @@ services:
         labels:
             traefik.docker.network: "inverseproxy_shared"
             traefik.enable: "true"
+
+            # Labels only for Traefik v1
             traefik.frontend.passHostHeader: "true"
             traefik.longpolling.port: "8072"
             traefik.port: "8069"
 
+            # Labels only for Traefik v2
+            traefik.entryPoints.web-secured.address: ":443"
+            traefik.entryPoints.web.address: ":80"
+            traefik.http.middlewares.web2web-secured.redirectscheme.scheme: "https"
+            traefik.http.routers.longpolling.entrypoints: "web-secured"
+            traefik.http.routers.longpolling.service: "longpolling"
+            traefik.http.routers.web-secured.entrypoints: "web-secured"
+            traefik.http.routers.web-secured.service: "odoo"
+            traefik.http.routers.web-secured.tls.certresolver: "main-acme"
+            traefik.http.routers.web.entrypoints: "web"
+            traefik.http.routers.web.middlewares: "web2web-secured"
+            traefik.http.routers.web.service: "dummy"
+            # HACK https://github.com/containous/traefik/issues/4863#issuecomment-491453743
```

Comment:
Have you considered this? containous/traefik#4863 (comment)

Reply:
Isn't it the same as I did?

Reply:
Sorry, wrong link, I meant https://github.com/containous/traefik/issues/4863#issuecomment-534065126

Reply:
I didn't try. Though, it should make configuration cleaner.